tmt/cli.py

```diff
@@ -53,31 +74,32 @@ def get_command(self, context, cmd_name):
             return click.Group.get_command(self, context, matches[0])
         context.fail('Did you mean {}?'.format(
             listed(sorted(matches), join='or')))
+        return None
 
 
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 #  Common Options
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-def verbose_debug_quiet(function):
```

Comment:
Could these options functions be improved to `def verbose_debug_quiet(function: Callable[..., Any]) -> Callable[..., Any]`?

Reply:
thanks, will check

Reply:
changed